pulsar-client-shaded/pom.xml

```diff
@@ -268,6 +268,21 @@
         </executions>
       </plugin>
 
+      <plugin>
+        <groupId>com.github.spotbugs</groupId>
+        <artifactId>spotbugs-maven-plugin</artifactId>
+        <version>${spotbugs-maven-plugin.version}</version>
```

Comment:
The version should have already been pinned in the `pluginManagement` section of the top-level pom

Reply:
Ok, I will delete this version configuration.